com.github.btpf.alexandria.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "com.github.btpf.alexandria",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "command": "alexandria",
+  "rename-icon": "alexandria",
+  "separate-locales": false,
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=/run/media"
+  ],
+  "modules": [
+    {
+      "type": "file",
```

Comment:
Invalid property

Reply:
Is it?
https://docs.flatpak.org/en/latest/manifests.html?highlight=modules#modules:~:text=file%20%2D%20local/remote%20files%20(these%20are%20copied%20into%20the%20source%20directory)

If not valid, what would be a proper alternative?

Reply:
It's for `sources`, see man flatpak-manifest

Reply:
Oh that flew by me twice. Will remove, Thanks.